mappings/net/minecraft/world/EntityView.mapping

```diff
@@ -76,19 +76,39 @@ CLASS net/minecraft/class_1924 net/minecraft/world/EntityView
 		ARG 1 entityClass
 		ARG 2 box
 		ARG 3 predicate
-	METHOD method_8333 getEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+	METHOD method_8333 getSurroundingEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+		COMMENT Computes a list of entities surrounding the given entity within some box.
+		COMMENT
+		COMMENT @return a list of entities within a box, excluding the given entity, all satisfying the given predicate
 		ARG 1 except
+			COMMENT the entity the box logically surrounds. This entity is ignored if it is inside the box.
 		ARG 2 box
+			COMMENT the box in which to search for entities
 		ARG 3 predicate
-	METHOD method_8335 getEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;)Ljava/util/List;
+			COMMENT a predicate which entities must satisfy in order to be included in the returned list.
+	METHOD method_8335 getSurroundingEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;)Ljava/util/List;
+		COMMENT Computes a list of entities surrounding the given entity within some box.
+		COMMENT
+		COMMENT @return a list of entities within a box, excluding the given entity
+		COMMENT @see #getSurroundingEntities(Entity, Box, Predicate)
 		ARG 1 except
+			COMMENT the entity the box logically surrounds. This entity is ignored if it is inside the box.
 		ARG 2 box
+			COMMENT the box in which to search for entities
 	METHOD method_8338 (Lnet/minecraft/class_1297;Lnet/minecraft/class_1297;)Z
 		ARG 1 e
-	METHOD method_8390 getEntities (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+	METHOD method_8390 getEntitiesByClass (Ljava/lang/Class;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+		COMMENT Computes a list of entities within some box whose runtime Java class is a subclass of the given type.
+		COMMENT
+		COMMENT @return a list of entities within the box whose runtime class is a subclass of the given class
+		COMMENT @throws IllegalArgumentException if {@code entityClass} is an interface or is not a subclass
+		COMMENT of {@link Entity}
 		ARG 1 entityClass
+			COMMENT the class returned entities must extends or implement
```

Comment:
```suggestion
			COMMENT the class the list of entities must extend
```

The docs say the game with throw if it is not a subclass of entity, which an interface cannot be.